app/models/like.rb

```diff
@@ -7,7 +7,7 @@ class Like < ApplicationRecord
   after_save :update_likes_count
   def update_likes_count
     post = self.post
-    post.likes_count += 1
+    post.likes_count = post.likes.count
     post.save
   end
```

Comment:
## Optional
- [ ] After adding Rails call back you can make that method  `private`

Reply:
- [ ] Please add likes associations as well, remember `like`
```ruby
belongs_to :user
belongs_to :post
```

Reply:
It was already added